corehq/apps/case_search/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+from corehq.apps.case_search.models import CaseSearchQueryAddition
+
+class CaseSearchQueryAdditionAdmin(admin.ModelAdmin):
+    list_display = ('name', 'id')
```

Comment:
Is there a way to show a link to a) the ES "[combining queries](https://www.elastic.co/guide/en/elasticsearch/guide/current/bool-query.html)" page, and 2) the spec for this feature in the admin page? Might be nice for future wanderers.